### ISSUE TITLE
issue with absolute address being used instead of relative

### ISSUE THREAD

**Issue by jeffcpina** [reonZ/pf2e-toolbelt#55]
I am using the freddy foundry docker setup and my modules are set up under the /Data/modules subfolder and not the /data/Data/modules subfolder for this reason the module shows an error on the console. I was able to circumvent the issue by adding a soft link that fools the main.js script but this does not work for the lang file.
Hope, this is the right place to report this issue. If not let me know.

**Comment by reonZ** [comment]
I dunno what your issue is but this isn't related to the module.
Literally every module do it the same way, all the paths in the `module.json` are relative so it doesn't matter what where your data is.

**Comment by jeffcpina** [comment]
First, thanks for your quick response. It is an odd behavior, what I did was copy the module to a new folder path "data/Data/modules" and everything started to work fine. I had the same issue with pf2e-workbench. What is funny is that once I was able to get this to work I did not need the new folder any longer. I removed the folder, rebooted the system container, and refreshed my browser cache. Everything is still working. What brought it to my attention was when I originally started using it and it was not working, I looked at the console it said "data/Data/pf2e-toolbelt/templates/betterMerchant/sheet.hbs not found". At any rate, we will just chalked this to a system glitch? 
Thanks again.
